src/main/java/mate/academy/springboot/datajpa/dto/mapper/CategoryMapper.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.springboot.datajpa.dto.mapper;
+
+import mate.academy.springboot.datajpa.dto.CategoryRequestDto;
+import mate.academy.springboot.datajpa.dto.CategoryResponseDto;
+import mate.academy.springboot.datajpa.model.Category;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CategoryMapper {
```

Comment:
let's create interface which will represents our mapper and implement it for category mapper and product mapper

Reply:
I created an interface and parametrized it with 3 generic values, since we have 3 different parameters used in our implementation: ProductResponseDto, ProductRequestDto, Product for the Product, and the same but with Category- prefix for the Category. This is kind of tricky implementation and I'm not sure if it makes the code better.
Or did you mean creating two different interfaces for product mapper and for category mapper? Because I understood it so that there should be the one interface and two implementations.